referral/forms.py

```diff
@@ -18,20 +19,23 @@ def __init__(self, referral_location_qs, *args, **kwargs):
         self.helper = FormHelper(self)
         self.fields['location'].widget = forms.widgets.CheckboxSelectMultiple()
         self.fields['location'].queryset = referral_location_qs
-        self.helper.add_input(Submit('submit', 'Create referral'))
+        self.helper.add_input(Submit('submit', _('Create referral')))
 
 
 class FollowupRequestForm(ModelForm):
     class Meta:
         model = models.FollowupRequest
         fields = ['due_date', 'contact_instructions']
         widgets = {'due_date': DateTimePicker(
-            options={"format": "MM/DD/YYYY"})}
+            options={"format": _("MM/DD/YYYY")})}
```

Comment:
When you translate this, do you change it to European style ("DD/MM/YYYY")? If so, internal logic will likely need to change.